vopr.bat

```diff
@@ -0,0 +1,25 @@
+@echo off
+
+:: Install Zig 0.8.0 if a zig folder does not already exist:
+if not exist zig\ (
+    call scripts/install_zig.bat 0.8.0
+    echo Running the TigerBeetle VOPR for the first time...
+    echo Visit https://www.tigerbeetle.com
+)
+
+:: If a seed is provided as an argument then replay the seed, otherwise test 1,000 seeds:
+if not "%~1"=="" (
+    :: Build in fast ReleaseSafe mode if required, useful where you don't need debug logging:
+    if "%~2"=="-OReleaseSafe" (
+        echo Replaying seed %~1 in ReleaseSafe mode...
+        call zig run src/simulator.zig -OReleaseSafe -- %~1

```

Comment:
Change to; `call zig\zig run src\simulator.zig -OReleaseSafe -- %~1`
We need to point to the `zig` instance that was just installed `(0.8.0)`.

## Otherwise
![2Capture](https://user-images.githubusercontent.com/1992108/134458975-51d4dfde-30eb-41f4-a7dd-5acec923444d.PNG)